frontend/src/reducers/property.js

```diff
@@ -2,34 +2,39 @@ const initialState = {
   error: "",
   fields: {
     id: "",
-    name: "",
+    landlordId: "",
+    navigatorId: "",
+    propertyName: "",
     address: "",
-    unitNum: "",
-    floorNum: "",
+    unitAptNum: "",
+    floor: 0,
     zipCode: "",
-    type: "",
-    numBeds: 0,
-    numBaths: 0,
-    numBathsShared: 0,
-    basement: false,
-    schoolDist: "",
-    busStop: false,
-    wheelchairAcc: false,
+    housingType: "",
+    bedrooms: 0,
+    bathrooms: 0,
+    sharedBathrooms: 0,
+    hasBasement: false,
+    schoolDistrict: "",
+    busLine: false,
+    wheelchairAccessibility: false,
     elevator: false,
-    dateAvail: "",
-    monthsAvail: 0,
-    rentPerMonth: 0,
-    appFee: 0,
+    dateFirstAvailable: new Date().toString(),
+    yearAvailable: new Date().getFullYear(),
+    potentialMonthAvailable: 0,
+    listingDate: new Date().toString(),
+    whereListed: "",
+    contactMethod: [],
+    lastContactDate: new Date().toString(),
+    lastContactedBy: "",
+    monthlyRent: 0,
+    applicationFee: 0,
     deposit: 0,
-    lastMonthRent: false,
-    CSRAP: false,
-    HVC: false,
-    MSV: false,
-    voucher4: false,
-    voucher5: false,
-    creditCompany: "",
-    backgroundCompany: "",
-    cleanRecord: false,
+    lastMonthRentRequired: false,
+    voucherTypeAccepted: [],
+    voucherTypeNotAccepted: ["MSV", "HCV", "CSRAP", "Voucher 4", "Voucher 5"],
```

Comment:
these need to be two separate fields and aren't co-dependent-- just because someone forgets to check the voucher accepted box doesn't mean that vouchers are not accepted.